modules/store/src/Cache/Context/StoreCacheContext.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+/**
+ * @file
+ * Contains \Drupal\commerce_store\Cache\Context\StoreCacheContext.
+ */
```

Comment:
The `@file` docblocks for files that have only one class/interface can be removed, so let's remove this.